core/_tables.scss

```diff
@@ -1,24 +1,29 @@
 table {
   border-collapse: collapse;
-  margin: $small-spacing 0;
+  margin: $base-spacing 0;
   table-layout: fixed;
+  text-align: left;
   width: 100%;
 }
 
-th {
-  border-bottom: 1px solid shade($base-border-color, 25%);
-  font-weight: 600;
-  padding: $small-spacing 0;
-  text-align: left;
+thead {
+  line-height: $heading-line-height;
+  vertical-align: bottom;
 }
 
-td {
+tbody {
+  vertical-align: top;
+}
+
+tr {
   border-bottom: $base-border;
-  padding: $small-spacing 0;
 }
 
-tr,
-td,
 th {
-  vertical-align: middle;
+  font-weight: 600;
+}
+
+th,
+td {
+  padding: $small-spacing $small-spacing $small-spacing 0;
```

Comment:
This is bad guidance. The best way to achieve this `padding` is via that shorthand (very long-hand of `margin-top`, `margin-right`, etc.). Just because I need `0` for `margin-left` doesn’t mean I should be throwing off our linter. I’ve adjust scss-lint (and Hound in return) in ca1aea7.